simple/kubernetes-apim-analytics/worker/wso2apim-analytics-worker-deployment.yaml

```diff
@@ -0,0 +1,95 @@
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2am-pattern-1-analytics-worker-deployment
+  namespace: wso2
+spec:
+  replicas: 1
+  minReadySeconds: 75
```

Comment:
@NishikaDeSilva shall we remove the `minReadySeconds` attribute, as per the explanation in [1], in all Kubernetes Deployments? IMO, since we don't have multiple containers within a Kubernetes Pod in this deployment and we can consider the Pod to be ready to serve requests, once the Kubernetes Readiness Probes are passed, successfully.

Applies to all Deployment definitions.

[1]: [How does minReadySeconds affect readiness probe?](https://stackoverflow.com/questions/53239081/how-does-minreadyseconds-affect-readiness-probe)